docs/codeql/reusables/supported-versions-compilers.rst

```diff
@@ -16,6 +16,7 @@
    .NET Core up to 3.1
 
    .NET 5, .NET 6, .NET 7, .NET 8, .NET 9","``.sln``, ``.csproj``, ``.cs``, ``.cshtml``, ``.xaml``"
+   GitHub Actions [12]_,"Not applicable",Not applicable,"``.github/workflows/*.yml``, ``.github/workflows/*.yaml``, ``action.yml``, ``action.yaml``"
```

Comment:
Is the path filtering not recursing inside the folder? For composite actions it is quite common to have them in subfolders.

Reply:
It does - the filter implementation uses `**/action.yml` and `**/action.yaml`. Maybe I should just write that. I didn't include it because we didn't seem to do that for the other language files, but it's confusing without it.